src/js/main.js

```diff
@@ -0,0 +1,28 @@
+let sortTable = document.getElementById('sortTable');
+
```

Comment:
need a link for gh-page
you copy this solution from - https://next.plnkr.co/edit/xoqlOD7IfVOeriG0L4po?p=preview&preview

Reply:
I have changed the solution. Thank you